homeassistant/components/recorder/migration.py

```diff
@@ -64,6 +64,7 @@ def _apply_update(engine, new_version):
         # Create indexes for states
         _create_index(engine, "states", "ix_states_last_updated")
         _create_index(engine, "states", "ix_states_entity_id_created")
+        _create_index(engine, "states", "ix_states_created_domain")
```

Comment:
This won't work. You need to create a new migration version. Otherwise people that already had this migration (=everyone) will not get it applied.